sys/dev/amdsmb/amdsmb.c

```diff
@@ -578,6 +578,7 @@ static driver_t amdsmb_driver = {
 };
 
 DRIVER_MODULE(amdsmb, pci, amdsmb_driver, amdsmb_devclass, 0, 0);
+PCI_PNP_INFO(amdsmb_devs);
```

Comment:
this looks like it's missing the other amdsmb changes needed. there's no amdsmb_devs here yet.

Reply:
Fixed it now. Thought the other part was already in the base.